docker/c7n-left

```diff
@@ -0,0 +1,34 @@
+FROM cgr.dev/chainguard/python:3.11-dev as builder
+
+ARG POETRY_VERSION="1.4.0"
+
+WORKDIR /app
+
+RUN python -m venv $HOME/tools && \
+    source $HOME/tools/bin/activate && \
+    pip3 install poetry==${POETRY_VERSION}
+
+ADD tools/c7n_left /app
+
+RUN ~/tools/bin/poetry export -o requirements.txt
+RUN pip3 install --user -r requirements.txt
+RUN pip3 install --user c7n
```

Comment:
ideally we'd install this out of repo, so it works the same as the other images, as opposed to picking latest release extant.